persistence/models/user.go

```diff
@@ -10,11 +10,12 @@ import (
 
 // User is used by pop to map your users database table to your go code.
 type User struct {
-	ID        uuid.UUID `db:"id" json:"id"`
-	Email     string    `db:"email" json:"email"`
-	Verified  bool      `db:"verified" json:"verified"`
-	CreatedAt time.Time `db:"created_at" json:"created_at"`
-	UpdatedAt time.Time `db:"updated_at" json:"updated_at"`
+	ID          uuid.UUID            `db:"id" json:"id"`
+	Email       string               `db:"email" json:"email"`
+	Verified    bool                 `db:"verified" json:"verified"`
+	Credentials []WebauthnCredential `has_many:"webauthn_credentials" json:"credentials,omitempty"`
```

Comment:
I think we should rename that to `WebauthnCredentials`  and `webauthn_credentials`, because it only contains WebauthnCredentials and no other like password. It would make it clearer, what credentials are returned.